### PR TITLE
HDDS-6478. EC: Fix ISA-l load hadoop native lib UnsatisfiedLinkError

### PR DESCRIPTION
**Pull request by cchenax**
## What changes were proposed in this pull request?
fix isa-l load hadoop native lib UnsatisfiedLinkError

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-6478

## How was this patch tested?
ci

### CODE REVIEW

**Comment by cchenax** [comment]
> You could also remove {private long nativeCoder;} from AbstractNativeRawEncoder file as well. Thanks

ok，thank you very much

File: hadoop-hdds/erasurecode/src/main/java/org/apache/ozone/erasurecode/rawcoder/NativeRSRawEncoder.java [hunk 1]
```diff
@@ -29,47 +33,35 @@
 @InterfaceAudience.Private
 public class NativeRSRawEncoder extends AbstractNativeRawEncoder {
 
-  static {
-    ErasureCodeNative.checkNativeCodeLoaded();
-  }
+  private org.apache.hadoop.io.erasurecode.rawcoder.NativeRSRawEncoder
+      hadoopNativeRSRawEncoder;
```
Comment (umamaheswararao) [inline hadoop-hdds/erasurecode/src/main/java/org/apache/ozone/erasurecode/rawcoder/NativeRSRawEncoder.java:37]: Are we using this LOG variable?

File: hadoop-hdds/erasurecode/src/main/java/org/apache/hadoop/io/erasurecode/rawcoder/HadoopNativeECAccessorUtil.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ */
+package org.apache.hadoop.io.erasurecode.rawcoder;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import org.apache.hadoop.hdds.annotation.InterfaceAudience;
+
+/**
+ * the class of Ozone Native EC.
```
Comment (umamaheswararao) [inline hadoop-hdds/erasurecode/src/main/java/org/apache/hadoop/io/erasurecode/rawcoder/HadoopNativeECAccessorUtil.java:26]: Repeated comment: Please change the doc as follows:
"This class is used to access some of the protected APIs from the Hadoop native EC java code. This is needed until we adapt the native ec code from the Hadoop."

**Review by umamaheswararao — COMMENTED** [review]
Thanks for the update. There are still few minor comments to address. Thanks

File: hadoop-hdds/erasurecode/src/main/java/org/apache/hadoop/io/erasurecode/rawcoder/HadoopNativeECAccessorUtil.java (hunk 2 above)
Comment (cchenax) [reply hadoop-hdds/erasurecode/src/main/java/org/apache/hadoop/io/erasurecode/rawcoder/HadoopNativeECAccessorUtil.java:26]: ok

**Review by umamaheswararao — APPROVED** [review]
LGTM, pending CI

**Pull request merged**